/**:
  ros__parameters:
    # --- 基本設定 ---
    speed_scale:          1.0  
    steer_scale:          1.0
    timer_hz:             50.0
    joy_timeout_sec:      0.5 

    # --- モード切替ボタン ---
    joy_button_index:     2   # △ボタンで手動モード
    ack_button_index:     3   # □ボタンで自律モード
    awsim_button_index:   1   # ○ボタンでAWSIMモード
    reset_button_index:   0   # ×ボタンでリセット

    # --- start/stop トリガボタン ---
    start_button_index:   7   # R2 ボタンで start=true
    stop_button_index:    6   # L2 ボタンで start=false

    # --- 十字キー（D-pad）軸インデックス ---
    dpad_lr_axis_index:   6   # 左右 (axes[6])
    dpad_ud_axis_index:   7   # 上下 (axes[7])

    reset_frame_id: "map"
    reset_pos_x: 89666.01577151686
    reset_pos_y: 43124.3307874416
    reset_pos_z: 0.0
    reset_ori_x: 0.0
    reset_ori_y: 0.0
    reset_ori_z: -0.9683930510846941
    reset_ori_w: 0.24942914547196962

## button index (DualShock4) humble

# 0: □
# 1: ×
# 2: ○
# 3: △
# 4: L1
# 5: R1
# 6: L2
# 7: R2
# 8: share
# 9: option
# 10: home
# 11: L3
# 12: R3

## axes index (DualShock4) for D-pad
# →: axes[6] == -1.0  → steer_offset +
# ←: axes[6] ==  1.0  → steer_offset –
# ↑: axes[7] ==  1.0  → speed_offset +
# ↓: axes[7] == -1.0  → speed_offset –
